src/modules/pair/services/pair.compute.service.ts

```diff
@@ -814,6 +910,30 @@ export class PairComputeService implements IPairComputeService {
         return addresses[farmAddressIndex];
     }
 
+    async getAllPairsFarmAddress(pairAddresses: string[]): Promise<string[]> {
+        const farmAddresses = farmsAddresses([FarmVersion.V2]).filter(
+            (address) => farmType(address) !== FarmRewardType.DEPRECATED,
+        );
+        const farmingTokenIds = await this.farmAbi.getAllFarmingTokenIds(
+            farmAddresses,
+        );
+        const allLpTokenIDs = await this.pairService.getAllLpTokensIds(
+            pairAddresses,
+        );
+
+        return allLpTokenIDs.map((lpTokenID) => {
+            const farmAddressIndex = farmingTokenIds.findIndex(
+                (tokenID) => tokenID === lpTokenID,
+            );
+
+            if (farmAddressIndex === -1) {
```

Comment:
here you used:
```
 if index === -1 { return undefined } else { return from array }
```
and in the function below you used:
```
return index === -1 { return undefined } else { return from array }
```

maybe keep consistency and choose a single way

Reply:
done